doc/en/Compiling-on-Linux.md

```diff
@@ -40,7 +53,7 @@ scons -h
 To install RHVoice execute the following command:
 
 ```bash
-# scons install
+sudo scons install
```

Comment:
someone may not use sudo I would like to leave this as before.

Reply:
done